phpBB/styles/prosilver/composer.json

```diff
@@ -0,0 +1,28 @@
+{
+	"name": "phpbb/phpbb-style-prosilver",
+	"description": "phpBB Forum Software default style",
+	"type": "phpbb-style",
+	"version": "3.3.0-a1-dev",
+	"homepage": "https://www.phpbb.com",
+	"license": "GPL-2.0",
+	"authors": [
+		{
+			"name": "phpBB Limited",
+			"email": "operations@phpbb.com",
+			"homepage": "https://www.phpbb.com/go/authors"
+		}
+	],
+	"support": {
+		"issues": "https://tracker.phpbb.com",
+		"forum": "https://www.phpbb.com/community/",
+		"wiki": "https://wiki.phpbb.com",
+		"irc": "irc://irc.freenode.org/phpbb"
+	},
+	"require": {
+		"php": ">=5.3.3"
```

Comment:
While not mandatory for styles, the minimum recommended [version](https://github.com/phpbb/phpbb/blob/master/phpBB/docs/INSTALL.html#L150) should be used.

Reply:
Do we really need this for styles? If this can maybe be `phpbb` instead it would probably be more useful.

Reply:
Agreed and removed.